src/auth/crypto.cpp

```diff
@@ -38,5 +69,59 @@ bool VerifyPassword(const std::string &password, const std::string &hash) {
   }
   return ret == 0;
 }
+}  // namespace BCrypt
+
+namespace SHA {
+std::string EncryptPassword(const std::string &password, const uint64_t number_of_iterations) {
+  unsigned char hash[SHA256_DIGEST_LENGTH];
+
+  SHA256_CTX sha256;
+  SHA256_Init(&sha256);
```

Comment:
It's deprecated to ensure backwards compatibility with older linux distributions. I don't have that functionality since newer version of openssl is not supported